setup.cfg

```diff
@@ -53,7 +100,7 @@ edit_on_github = True
 github_project = lcogt/banzai
 
 # version should be PEP440 compatible (http://www.python.org/dev/peps/pep-0440)
-version = 0.28.5
+version = 0.29.0
 
 [options]
 setup_requires =
```

Comment:
Should we specify (and pin) dependencies here, rather than in the dockerfile?

Reply:
I think we should just specifiy them here an not pin then unless there is specific need.

Reply:
Don't we already specify them here?